src/queries/queries.js

```diff
@@ -504,3 +504,14 @@ export const buildAccountQueryByLogin = login => ({
     as: `${ACCOUNTS_DOCTYPE}/account_type/login/${login}`
   }
 })
+
+export const buildFirstTimeserieQuery = () => {
+  return {
+    definition: Q('io.cozy.timeseries.geojson')
+      .where({ startDate: { $gt: null } })
+      .select(['_id', 'startDate'])
+      .indexFields(['startDate'])
+      .sortBy([{ startDate: 'asc' }])
+      .limitBy(1)
+  }
+}
```

Comment:
On risque de rencontrer un souci ici, à confirmer avec @paultranvan
cf: https://github.com/cozy/coachCO2/pull/391#discussion_r1482756169

Reply:
The simplest is to re-use this same query https://github.com/cozy/coachCO2/pull/388#discussion_r1484132644